src/ordering.c

```diff
@@ -541,6 +542,7 @@ enum order_difference_reason compare_nodes_refpolicy_generic(struct ordering_met
 			if (first->data.d_data->flavor != second->data.d_data->flavor) {
 				CHECK_FLAVOR_ORDERING(d_data, DECL_BOOL, ORDER_DECLARATION_SUBSECTION);
 				CHECK_FLAVOR_ORDERING(d_data, DECL_ATTRIBUTE, ORDER_DECLARATION_SUBSECTION);
+				CHECK_FLAVOR_ORDERING(d_data, DECL_ATTRIBUTE_ROLE, ORDER_DECLARATION_SUBSECTION);
```

Comment:
The style guide doesn't specify the order of attribute_role declarations in the declarations section.  I think this can just be omitted.

Reply:
dropped